Guess game/Program.cs

```diff
@@ -51,39 +56,60 @@ static int StartGame()
                 foreach (KeyValuePair<string, string> riddle in Riddles)
                 {
                     int guessCount = 0;
+                    Console.WriteLine("------------------------------------------------------------------------------");
                     Console.WriteLine(riddle.Key);
 
                     while (guessCount < guessLimit)
                     {
-                        Console.Write("Enter your guess ");
+                       Console.WriteLine();
+                       Console.Write("Įveskite savo spėjimą: ");
                        string guess = Console.ReadLine();
 
                         if (guess == riddle.Value)
                         {
-                            GiveAnswer(correct: true);
+                            GiveAnswer(correct: true, lastAttempt: false);
                             progress += guessLimit - guessCount;
                             break;
                         }
 
                         if (guessCount == guessLimit - 1 & guess != riddle.Value)
                         {
-                            GiveAnswer(correct: false);
+                            GiveAnswer(correct: false, lastAttempt: true);
                             break;
                         }
 
+                        GiveAnswer(correct: false, lastAttempt: false);
                         guessCount++;
                     }
                 }
 
             return progress;
         }
 
-        private static void GiveAnswer(bool correct)
+        private static void GiveAnswer(bool correct, bool lastAttempt)
         {
-            Console.WriteLine(correct ? "Nice you get it " : "You did not get it ");
+            if (correct == true && lastAttempt == false)
```

Comment:
if statements can be simplified. instead of now:
```
if (correct == true && lastAttempt == false)
{
  ...
}
else if (correct == false && lastAttempt == false)
{
  ...
}
else if (correct == false && lastAttempt == true)
{
  ...
}
```  
it can be simplified to this:
```
if(correct)
{
  ...
}
else if (!lastAttempt)
{
  ...
}
else {
  ...
}
```
program will work in the same way, but IF will be full (having ELSE condition also) it means it never result in case where we don't have logic written & for booleans `correct == true` can be shorten as `correct` ir will be the same or for negative `lastAttempt == false` can be shorten to `!lastAttempt` (! mark in front of boolean)